pkg/apis/resources.cattle.io/v1/types.go

```diff
@@ -112,6 +112,9 @@ type RestoreSpec struct {
 	Prune                      *bool            `json:"prune"` //prune by default
 	DeleteTimeoutSeconds       int              `json:"deleteTimeoutSeconds,omitempty"`
 	EncryptionConfigSecretName string           `json:"encryptionConfigSecretName,omitempty"`
+
+	// When set to true, controller will ignore any error during restore CRs and CRDs and mark it as done
```

Comment:
I believe we're ignoring any restore errors in this case, not just CRs and CRDs, so this comment is a bit misleading.
```suggestion
	// When set to true, the controller ignores any errors during the restore process
```